praw/models/reddit/mixins/inboxtoggleable.py

```diff
@@ -7,6 +7,10 @@ class InboxToggleableMixin:
 
     def disable_inbox_replies(self):
         """Disable inbox replies for the item.
+        
+        .. note::
+        
+            This can only apply to items created by the authenticated user.
```

Comment:
```suggestion

        .. note::

            This can only apply to items created by the authenticated user.
```
Look like there is trailing whitespace that is causing issues.